cli/command/signup/cmd_test.go

```diff
@@ -0,0 +1,3 @@
+package signup
+
+// TODO: Tests!
```

Comment:
I'm not quite sure what degree of testing we want.

I could just have a test to call the command and that it ejects the same hardcoded output? Or perhaps just that somewhere in the output is the bit.ly link?

Reply:
Depends on how crazy you wanna go with it :D
Imho, there is little to break with this one liner.
Otherwise you can always create a separate method that prints this signup message to a writer (via `fmt.Fprintln()`).

E.g.
```
func printSignup(w io.Writer) {
  fmt.Fprintln(w, "my signup msg")
}
```

Given a custom read-writer, you could then verify that it did what it was supposed to do (rather than test the actual command). 🤷 Up to you, a bit overkill for a one liner, but that'd be the "clean" way of doing it.

Reply:
I'm going to err on less is more here and just omit tests👌 If this command someday does more than print a canned string it'll get supporting tests.